account-kit/smart-contracts/src/coinbase/tests/coinbase-account.test.ts

```diff
@@ -0,0 +1,144 @@
+import {
```

Comment:
Looks good overall! Could we move this to `packages/smart-accounts/`, since this pattern is building on the v5 smart account concepts?